examples/jsm/loaders/3MFLoader.js

```diff
@@ -359,8 +359,7 @@ class ThreeMFLoader extends Loader {
 				const colorNode = colorNodes[ i ];
 				const color = colorNode.getAttribute( 'color' );
 
-				colorObject.setStyle( color.substring( 0, 7 ) );
-				colorObject.convertSRGBToLinear(); // color is in sRGB
+				colorObject.setStyle( color.substring( 0, 7 ), SRGBColorSpace );
```

Comment:
Note: SRGBColorSpace is the default — this is just to be explicit that we know the loader's colors are sRGB. Would also be fine with removing it. See next comment.

Reply:
I'm concerned that down the road (a) our understanding of a format's specification may change, or (b) our approach to input color space may change. For example, our current defaults...

```js
color.setRGB( r, g, b, ColorManagement.workingColorSpace );
color.setHex( hex, SRGBColorSpace );
```

... could become configurable ...

```js
color.setRGB( r, g, b, ColorManagement.workingColorSpace );
color.setHex( hex, ColorManagement.inputColorSpace );
```

Reply:
In other words, the loader's reasons for using this value are distinct from our reasons for using it as the default in the setter. @gkjohnson also seems to prefer something explicit. Perhaps this is a good compromise in a loader?

```js
const COLOR_SPACE_3MF = THREE.SRGBColorSpace;

...

colorObject.setStyle( color.substring( 0, 7 ), COLOR_SPACE_3MF );
```